extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchAggregatorTest.java

```diff
@@ -112,6 +116,34 @@ public void buildSketchesAtIngestionTime() throws Exception
     Assert.assertEquals(200, (double) row.get(0), 0.1);
   }
 
+  @Test
+  public void buildSketchesAtIngestionTimeMultiValueWithTransformations() throws Exception
+  {
+    Sequence<ResultRow> seq = helper.createIndexAndRunQueryOnSegment(
+        new File(this.getClass().getClassLoader().getResource("hll/hll_raw.tsv").getFile()),
+        buildParserJson(
+            Collections.singletonList("dim"),
+            Arrays.asList("timestamp", "dim", "multiDim", "id"),
+            ImmutableList.of(
+                new ExpressionTransform(
+                    "multiDimAppended",
+                    "array_append(multiDim, '15')",
+                    ExprMacroTable.nil()
```

Comment:
@clintropolis yes, I guess you're right.

However I am still not able to make this test pass. Is there a simple way to run these tests?

Or maybe the issue is evident right away? I haven't found any test with transformations, I suppose this test implementation is flawed.